packages/cyberstorm/src/components/AntiResult/AntiResultIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import styles from "./AntiResult.module.css";
+import { Icon } from "../Icon/Icon";
+import { classnames } from "../../utils/utils";
+
+interface Props {
```

Comment:
Why not PropsWithChildren?

Reply:
Because the children have to be JSX.Elements aka FontAwesomeIcons

Reply:
In that case you could consider [something like this](https://github.com/thunderstore-io/thunderstore-ui/blob/master/packages/cyberstorm/src/components/NewTabs/Tabs.tsx#L98) to actually force the content. But this is fine as it is as well.